lms/static/sass/elements/_controls.scss

```diff
@@ -403,3 +403,22 @@
       margin-bottom: none;
     }
   }
+
+  .btn-link {
```

Comment:
@marcotuts I've added new sass for btn-link. Can you please give it a quick look. Thanks.